finality-aleph/src/sync/substrate/mod.rs

```diff
@@ -20,6 +22,25 @@ pub use status_notifier::SubstrateChainStatusNotifier;
 pub use translator::Error as TranslateError;
 pub use verification::{SessionVerifier, SubstrateFinalizationInfo, VerifierCache};
 
+impl BlockImport<Block> for Box<dyn ImportQueueService<Block>> {
+    fn import_block(&mut self, block: Block) {
+        let origin = BlockOrigin::NetworkBroadcast;
+        let incoming_block = IncomingBlock::<Block> {
```

Comment:
if we have no idea if this is correct let's wait with this pr and create a task for discovery and research.

Reply:
Its not about aura here but a block import, right? Isn't it the case that it might work in e2e test but we wont know why? Also, it wont tell us about the subtle meaning of some of these args.

Reply:
I think that the actual verification is done by aura? Nevertheless, it's infeasible to check this here, I'd still suggest waiting for A0-2406 (it's the same epic).
> Isn't it the case that it might work in e2e test but we wont know why?
yes, but I do hope that our tests cover everything that we need to check...

Reply:
Still, imo we need to know what is going on here. Even if this blocks another task it just mean it has at least the same priority as the blocked one

Reply:
ok, I'll read the substrate code today, and try to figure sth out

Reply:
OR we could simply merge these two tasks - what do you think? I'd feel more comfortable with that, since I'd be able to, at least partially, check if I understand substrate api correctly

Reply:
Yep, that would be nice :)